libethereum/Client.h

```diff
@@ -198,6 +198,9 @@ class Client: public ClientBase, protected Worker
 	virtual Block block(h256 const& _block) const override;
 	using ClientBase::block;
 
+	/// should be called after the constructor of the most derived class finishes.
+	void afterConstructor() { startWorking(); };
```

Comment:
I would call this method `startWorking()`

Reply:
OK, I'll try that.